package.json

```diff
@@ -15,9 +15,19 @@
     "lint-staged": "turbo run lint-staged"
   },
   "devDependencies": {
+    "cpy-cli": "^3.1.1",
+    "rimraf": "^3.0.2",
     "turbo": "~1.2.4",
     "webpack": "5"
   },
+  "resolutions": {
```

Comment:
List is shorter.
You have removed those which are not needed?

Reply:
yup